locale/en/LC_MESSAGES/chapters/chapter-4.po

```diff
@@ -178,3 +182,107 @@ msgid ""
 "the workshop when not using OSGeoLive this extra steps are needed:"
 msgstr ""
 
+#~ msgid "PostgreSQL 9.4 or higher"
```

Comment:
Remove the saved previous translations of the file, they are already in transifex